docs/sources/enterprise/white-labeling.md

```diff
@@ -63,23 +63,29 @@ The configuration file in Grafana Enterprise contains the following options. Eac
 # Set to complete URL to override apple/ios icon
 ;apple_touch_icon =
 ```
-
-Below is an example for how to replace the default footer and help links with two custom links.
+You can replace the default footer links (Documentation, Support, Community) and even add your own custom links.
+Below is an example for how to replace the default footer and help links with new custom links.
 
 ```ini
-footer_links = support guides
+footer_links = support guides extracustom
 footer_links_support_text = Support
 footer_links_support_url = http://your.support.site
 footer_links_guides_text = Guides
 footer_links_guides_url = http://your.guides.site
+footer_links_extracustom_text = Custom Text
+footer_links_extracustom_url = http://your.custom.site
 ```
 
 Here is the same example using environment variables instead of the custom.ini or grafana.ini file.
 
 ```
-GF_WHITE_LABELING_FOOTER_LINKS=support guides
+GF_WHITE_LABELING_FOOTER_LINKS=support guides extracustom
 GF_WHITE_LABELING_FOOTER_LINKS_SUPPORT_TEXT=Support
 GF_WHITE_LABELING_FOOTER_LINKS_SUPPORT_URL=http://your.support.site
 GF_WHITE_LABELING_FOOTER_LINKS_GUIDES_TEXT=Guides
 GF_WHITE_LABELING_FOOTER_LINKS_GUIDES_URL=http://your.guides.site
+GF_WHITE_LABELING_FOOTER_LINKS_EXTRACUSTOM_TEXT=Custom Text
+GF_WHITE_LABELING_FOOTER_LINKS_EXTRACUSTOM_URL=http://your.custom.site
 ```
+
+> **Note:** Any link other than the Grafana edition and version/build number that is not specified in `footer_links` or `GF_WHITE_LABELING_FOOTER_LINKS` is removed from the footer.
```

Comment:
```suggestion
> **Note:** If a link, which contains the Grafana edition and version or build number, is specified in `footer_links` or `GF_WHITE_LABELING_FOOTER_LINKS`, it is present in the footer. If a link does not contain this information or is not specified by those parameters, it is not present in the footer.
```
The `version/build` bit is ambiguous; must a link contain either a version or build number, one of them (where here there are two terms for the same thing), or both?

Reply:
The point that I was trying to make clear is that the footer will always contain these two items: the grafana edition (as one link that goes to the licensing tab), and the version & build number (as another link that doesn't technically go anywhere)

They look like this:

![image](https://user-images.githubusercontent.com/11842311/112324116-aa383180-8c6f-11eb-9e0f-ea96629bf7ea.png)

So, a link doesn't have to contain these, my point is that if you specify `footer_links` or `GF_WHITE_LABELING_FOOTER_LINKS` it will remove all *other* default links except for these two.

Reply:
@oddlittlebird, got a second to provide feedback on this? The cognitive load to parse that previous version strikes me as too high.

Reply:
Consider rewriting this so that the included items are in a list rather than inline. 

Also, what you just wrote works pretty well, Eamon.

Reply:
Pushed some wording changes based on suggestions.